packages/toolkit/src/components/LatestChangesCard.tsx

```diff
@@ -0,0 +1,66 @@
+import { Icons } from "@instill-ai/design-system";
+import { useChangelogs } from "../lib/react-query-service/misc/useChangelogs";
+
+interface Changelog {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
+  date: string | null;
+  title: string;
+  published: boolean;
+}
+
+
+const LatestChangesCard: React.FC = () => {
+  const { data: changelogs, isLoading, isError } = useChangelogs();
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (isError) {
+    return <div>Error fetching changelogs.</div>;
+  }
+
+  return (
+    <div className="mt-4 flex flex-col gap-y-2 rounded-sm border border-semantic-bg-line p-4">
+      <h2 className="mb-4 text-2xl font-bold">Latest Changes</h2>
+      {changelogs?.map((changelog: Changelog) => (
+        <div key={changelog.id}>
+          <button
+            type="button"
+            className="my-auto flex-auto rounded-sm bg-semantic-accent-bg p-2 capitalize text-semantic-accent-default product-button-button-2 hover:!underline"
+          >
+            {changelog.date && new Date(changelog.date).getTime() !== 0
+              ? new Date(changelog.date).toLocaleDateString("en-US", {
+                year: "numeric",
+                month: "long",
+                day: "numeric",
+              })
+              : new Date(changelog.updatedAt).toLocaleDateString("en-US", {
+                year: "numeric",
+                month: "long",
+                day: "numeric",
+              })}
+          </button>
+          <p>{changelog.title}</p>
+        </div>
+      ))}
+      <a
+        href="https://instill-ai.productlane.com/changelog"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <button
+          type="button"
+          className="my-auto ml-2 mt-4 flex justify-start text-semantic-accent-default product-button-button-2 hover:!underline"
+        >
+          View changelog
+          <Icons.ChevronRight className="my-auto h-4 w-4 stroke-semantic-accent-default" />
+        </button>
+      </a>
```

Comment:
@thewbuk anchor element should not have interactive element inside, or this will violate W3C HTML protocol. You either remove the button and use div there, or use button directly and router.push

> The [a](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element) element can be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g., buttons or other links). This example shows how this can be used to make an entire advertising block into a link:

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element